python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -826,6 +826,29 @@ def ElementwiseMulTensorIntModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class ElementwiseMishModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+        self.mish = torch.nn.Mish()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.float32, True),
+    ])
+    def forward(self, x):
+        return self.mish(x)
```

Comment:
We can directly call the aten op instead of initializing the nn.module. I guess it's `torch.ops.aten.mish(x)`.

Reply:
Thanks for this - I was initially trying `torch.ops.aten.Mish(x)` hence wasn't able to get through.